test/21-supervisor-api.spec.ts

```diff
@@ -95,7 +95,6 @@ describe('SupervisorAPI', () => {
 				await request
 					.get('/v1/healthy')
 					.set('Accept', 'application/json')
-					.set('Authorization', `Bearer ${VALID_SECRET}`)
```

Comment:
The auth step is only applied after `ping` and `healthy` endpoint. We would also need the doc to correctly update the `ping` endpoint as it currently shows `apiKey` as required param.

Ref: https://github.com/balena-io/balena-supervisor/blob/master/docs/API.md#http-api-reference

Reply:
good catch. The documentation for the API is located in https://github.com/balena-io/balena-supervisor/blob/master/docs/API.md. Once that file is modified we can notify the production team to release it to the site. Feel free to include that change in this PR.